components/core/Dockerfile

```diff
@@ -1,19 +1,35 @@
-FROM python:3
+FROM python:3:alpine
 
 MAINTAINER SCoRe Lab Community <commuity@scorelab.org>
+LABEL description "uWSGI + Flask based on Alpine Linux"
+
 
 ARG BUILD_DATE
 ARG VCS_REF
 
 WORKDIR /home/Bassa
 EXPOSE 5000
 COPY . .
+EXPOSE 3031
+
+RUN apk add --no-cache \
+    uwsgi \
+    uwsgi-python3 \
+    && python3 -m ensurepip \
+    && rm -r /usr/lib/python*/ensurepip \
+    && pip3 install --upgrade pip setuptools \
+    && pip3 install -r requirements.txt \
+    && rm -r /root/.cache \
+    && ./setup.sh \
+    && python setup.py develop 
 
-RUN apt-get update \
-&& ./setup.sh \
-&& python setup.py develop 
+COPY uwsgi.ini /etc/uwsgi/
 
-CMD [ "python", "Main.py" ]
+CMD [ "uwsgi", "--socket", "0.0.0.0:3031", \
+               "--plugins", "python3", \
+               "--protocol", "uwsgi", \
+               "--wsgi", "main:application", \
+      "python", "Main.py"]
```

Comment:
Are we running `python Main.py` command again? Or are we passing them as params to uWSGI?